package.json

```diff
@@ -33,12 +48,33 @@
     "eslint-plugin-node": "^8.0.1",
     "eslint-plugin-prettier": "^3.0.1",
     "fixturify": "^0.3.4",
+    "husky": "^3.0.0",
     "mocha": "^5.2.0",
     "multidep": "^2.0.0",
     "prettier": "^1.16.4",
+    "pretty-quick": "^1.11.1",
     "rsvp": "^4.8.4"
   },
   "engines": {
-    "node": "6.* || 8.* || >= 10.*"
+    "node": "8.* || >= 10.*"
+  },
+  "files": [
+    "index.js",
+    "index.ts",
```

Comment:
Should we publish these `ts` files?  I think our consumers only need `js` `map` `d.ts`